engine/runtime/src/test/resources/Test_Simple_Imports/package.yaml

```diff
@@ -1,4 +1,4 @@
-name: TestSimpleImports
+name: Test_Simple_Imports
```

Comment:
Why do we need to fix the `name`?

Reply:
I wanted the name to match the module name. While we allow arbitrary names for the projects, using the module name for the project will help to discover it and be less confusing in general. The correct Enso-style module name, in this case, would be `Test_Simple_Imports`